frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -761,15 +784,56 @@ export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProp
   );
 });
 
+const AlertToCreateCustomResource: React.SFC<AlertToCreateCustomResourceProps> = (props) => {
+  const { cr } = props;
+
+  const crName = cr?.[0]?.kind;
+  const crDefinition = props.obj.spec?.customresourcedefinitions?.owned?.find(
+    (api) => api.kind === crName,
+  );
+  const crModelName = crDefinition?.name;
+  const { group: crGroup } = groupVersionFor(cr?.[0]?.apiVersion);
+  const { findModel } = useModelFinder();
+  const model = findModel(crGroup, crModelName.split('.').shift());
```

Comment:
We need to handle cases where the model is not found.

Reply:
`useK8sGet` will survive a null model value.  In that case we just continue to show the `Create` button.  What would you prefer?

Reply:
I'd expect the link to be broken when the resource doesn't exist yet on the cluster. I suspect we have this issue on the install operator status as well.

We should also use `useAccessReview` here to only show the alert to users who have authority to create the resource.

Reply:
For all the links, we use `referenceForGroupVersionKind` to build the URL, so if the reference does not exist, then we get 404.  We should probably discuss if there is anything we can do in those situations further, but preferably out of the scope of this PR.

Reply:
`useAccessReview` has been added.